### PR TITLE
Remove foreman_custom_parameters package

### PR DESCRIPTION
**Pull request by ekohl**
This package hasn't seen an update since 2014 and is archived on Github. In fact, even the 0.0.2 version on Rubygems was never pushed to Github.

### CODE REVIEW

**Comment by ekohl** [comment]
Also made a post about it: https://community.theforeman.org/t/removing-foreman-custom-parameters-from-nightly/16217

**Comment by tbrisker** [comment]
[test rpm]

**Comment by ekohl** [comment]
I don't think our CI handles package deletion well given how little we remove packages.

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -305,6 +306,9 @@ selinuxenabled && load_policy || :
 %{_root_sysconfdir}/rpm/macros.%{scl_name}-scldevel
 
 %changelog
+* Mon Nov 18 2019 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl>
```
Comment (tbrisker) [inline packages/foreman/tfm/tfm.spec]: This is failing linting due to missing version

Comment (ekohl) [reply packages/foreman/tfm/tfm.spec]: I wonder how obal messed up there. Fixing manually.

**Comment by ekohl** [comment]
:green_heart: now. Looking back at tfm I think the nodejs addition should have been version 5.1 or even 6.0. Just a release feels odd now.

**Pull request merged**